webapp/doc_parser.py

```diff
@@ -0,0 +1,234 @@
+# Standard library
+from canonicalwebteam.discourse_docs import DocParser
+
+import re
+
+# Packages
+import dateutil.parser
+import humanize
+from bs4 import BeautifulSoup, NavigableString
+from jinja2 import Template
+
+HEADER_REGEX = re.compile("^h[1-6]$")
+
+
+class FastDocParser(DocParser):
```

Comment:
@jkfran could you highlight in this file where you've changed stuff from the original parser?

Reply:
@nottrobin Here are the changes on the overrided methods:
https://editor.mergely.com/95GiOuZw/